### DISCUSSION TITLE
Connection string

### DISCUSSION THREAD

**Discussion by KevinBurton** [dbeaver/dbeaver#12760] [Q&A] [answered]
I go through all of the tabs and fill it out and then test connection and the connection says OK. I browse the MariaDB databases and tables and execute some queries. When I am done I would like to take my new found knowledge and access the database through EF. In order to do that I need to get a connection string that specifies the default database, the server, credentials (user name/password), and probably the provider. I failed to see any place that I can see what the connection string is being used. When I 'Test Connection' I briefly see what appears to be a connection string but in goes by too quickly for me to note it.  

**Describe the solution you'd like**
I would like for there to be a display of the connection string

**Describe alternatives you've considered**
I have considered going to MySQL or Pomelo but they don't offer a UI that I could see the connection string on

**Comment by uslss** [comment, marked as answer]
Its template is available in Driver settings. Does it solve your problem?
![image](https://user-images.githubusercontent.com/31996417/111607806-a593f900-87e9-11eb-9cff-043c368fda43.png)

**Comment by clockback** [comment]
It would still be preferable to see the connection string. It is not always easy to intuit how to construct such a string from a template which doesn't have _direct_ correspondence.

**Comment by kseniiaguzeeva** [comment]
Please open Properties to see connection information (click your connection in DB Navigator and select Window>Show view>Properties).

![Screenshot 2021-05-13 at 11 18 05](https://user-images.githubusercontent.com/51405061/118099061-6646ea80-b3dd-11eb-8f7c-54b04893ec7c.png)
